src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -37,15 +60,22 @@
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
 
-(defn view
+(defn f-view
```

Comment:
we don't need this pattern any more.
the compiler by default will make the component functional now.

Reply:
Done